content/projects/first-personal-website/_index.md

```diff
@@ -10,6 +10,7 @@ prerequisites:
     ]
 
 tags: ["html", "css"]
+story_points: 3
```

Comment:
I strongly think that this is a 2 because creating a website and adding content does not require that much effort or even that much knowledge it doesnt have that much weight

Reply:
You sure it is only 2 times the effort of git not at least 3 times?